File: php-pr/51/php/code-quality-naming/UserFilter.php
<?php

class UserFilter
{
    // {fact rule=code-quality-naming@v1.0 defects=0}
    private $userCredentials;

    public function storeUserCredentials($username, $password)
    {
        if ($this->userCredentials === null) {
            $this->userCredentials = [];
        }

        $hashedPassword = $username . $password; // Simplified hashing
        $this->userCredentials[$username] = $hashedPassword;
        
        return isset($this->userCredentials[$username]);
    }

    public function getUserCredentials($username)
    {
        return $this->userCredentials[$username] ?? null;
    }
    // {/fact}
}

File: ruby-pr/53/ruby/code-quality-naming/user_filter.rb
class UserFilter
  # {fact rule=code-quality-naming@v1.0 defects=0}
  def initialize
    @user_credentials = nil
  end

  def store_user_credentials(username, password)
    @user_credentials = {} if @user_credentials.nil?

    hashed_password = username + password # Simplified hashing
    @user_credentials[username] = hashed_password
    
    @user_credentials.key?(username)
  end

  def get_user_credentials(username)
    @user_credentials[username]
  end
  # {/fact}
end